ext/standard/tests/general_functions/proc_open_sockets1.phpt

```diff
@@ -1,5 +1,9 @@
 --TEST--
 proc_open() with output socketpairs
+--SKIPIF--
+<?php
+if (getenv("SKIP_SLOW_TESTS")) die("SKIP_SLOW_TESTS");
```

Comment:
The message must start with `skip ...` or it won't actually be skipped.

Reply:
Ah, actually it will, likely by accident. The regex is `/^skip\s*(.+)/i` instead of `/^skip\s+(.+)/i`. Anyway, please stick to the existing convention.

Reply:
>the existing convention

you sure there is one?

sapi/fpm/tests/reload-uses-sigkill-as-last-measure.phpt:
```
if (getenv('SKIP_SLOW_TESTS')) die('skip slow tests excluded by request');
```
tests/basic/timeout_variation_9.phpt:
```
if (getenv("SKIP_SLOW_TESTS")) die("skip slow test");
```
ext/phar/tests/bug13727.phpt:
```
<?php if (getenv('SKIP_SLOW_TESTS')) die('skip'); ?>
```
ext/sockets/tests/socket_shutdown-win32.phpt:
```
if (getenv("SKIP_SLOW_TESTS")) {
    die('skip: Slow test');
}
```

Reply:
Sure, we're inconsistent. That doesn't mean we need yet another variation 😉